src/doc/dev_manual/CI.rst

```diff
@@ -0,0 +1,29 @@
+Docker Containers For CI Testing
+=================================
+
+We use Azure Pipelines for CI testing Visit's Pull Requests.
+
+* `Ascent Azure DevOps Space <https://dev.azure.com/visit-dav/VisIt/>`_
+
+
+To speed up our CI testing we use Docker containers with pre-built third party libraries. These containers leverage our ``build_visit`` third party build  process. The Docker files and build scripts used to create these containers are in ``scripts/ci/docker``. To update the containers (assuming you have Docker installed):
+
+ * Run ``python build_docker_visit_ci.py`` to build and tag new version of our TPL container.
```

Comment:
Typo: "to build and tag new version" to "to build and tag the new version".

Reply:
thanks, fixed these typos